indexer-agent/run.sh

```diff
@@ -43,6 +44,7 @@ export INDEXER_AGENT_POSTGRES_USERNAME=postgres
 export INDEXER_AGENT_POSTGRES_PASSWORD=
 export INDEXER_AGENT_PUBLIC_INDEXER_URL="http://indexer-service-rs:${INDEXER_SERVICE}"
 export INDEXER_AGENT_TAP_SUBGRAPH_ENDPOINT="http://graph-node:${GRAPH_NODE_GRAPHQL}/subgraphs/semiotic/tap"
+export INDEXER_AGENT_MULTINETWORK_MODE=true
```

Comment:
This seems to error out why I try to run this branch, unless I set it to false.

Reply:
Ah yeah I'm mostly running with the run-override.sh script. MultiNetwork is going away.